Makefile

```diff
@@ -48,22 +48,31 @@ go-deps: $(GO_DEPS)
 go-deps-clean:
 	$(RM) -r vendor
 
-sonic-telemetry: $(GO_DEPS)
-	$(GO) install -mod=vendor $(BLD_FLAGS) github.com/Azure/sonic-telemetry/telemetry
-	$(GO) install -mod=vendor $(BLD_FLAGS) github.com/Azure/sonic-telemetry/dialout/dialout_client_cli
+sonic-gnmi: $(GO_DEPS)
+ifeq ($(CROSS_BUILD_ENVIRON),y)
```

Comment:
This PR description says that it changes `sonic-telemetry` into `sonic-gnmi`, so, could you limit changes to this and create anohter PR for introduction of other changes?